q2_diversity/_alpha/alpha_correlation_assets/src/render.js

```diff
@@ -17,7 +17,8 @@ export function render(svg, data) {
 
   const { xAxisLabel, yAxisLabel, minX, maxX, minY, maxY } = data;
 
-  const x = scaleLinear().domain([minX, maxX]).range([0, width]).nice();
+  // Add a ~3% padding to minX values, also handle for a minX of 0
+  const x = scaleLinear().domain([minX * 0.97 || minX - 0.03, maxX]).range([0, width]).nice();
```

Comment:
That one was just to make it non-zero after it fell back on the `||` for `minX` being 0.